settings/geo_coldbox.json

```diff
@@ -29,7 +29,7 @@
 	    "n_tot_channels":"1792",
 	    "sampling":"2",
 	    "n_sample":"8192",
-	    "ADC_to_fC":"1",
+	    "ADC_per_fC":"266.76",
```

Comment:
This change also should probably be merged with the commit where you rename the variable.